--- origsrc/xorg-cf-files-1.0.2/cygwin.cf	2005-07-06 04:47:49.000000000 -0500
+++ src/xorg-cf-files-1.0.2/cygwin.cf	2008-11-04 10:28:33.040801300 -0600
@@ -62,12 +62,23 @@
 # define OSTeenyVersion		DefaultOSTeenyVersion
 #endif
 
-#ifndef ProjectRoot
-# define ProjectRoot 		/usr/X11R6
+#ifndef XAppLoadDir
+# define XAppLoadDir			EtcX11Directory/app-defaults
+#endif
+#ifndef SystemManDirectory
+# define SystemManDirectory	/usr/share/man
+#endif
+#ifndef FontDir
+# define FontDir				/usr/share/fonts
+#endif
+#ifndef FontEncDir
+# define FontEncDir			/usr/share/fonts/encodings
 #endif
 
 #if !CrossCompiling
+#ifndef HasGcc2
 # define HasGcc2		YES
+#endif /* HasGcc2 */
 #endif /* !CrossCompiling */
 #define HasGcc2ForCplusplus	YES
 #define GccUsesGas		YES
@@ -98,15 +100,12 @@
  */
 #define Hastcl			YES
 #define Hastk			YES
-#define TclLibName		tcl80
-#define TkLibName		tk80
+#define TclLibName		tcl84
+#define TkLibName		tk84
 
 #define HasSockets		YES
 #define GzipFontCompression	YES
 #define HasZlib			YES
-#ifndef GzipLibrary
-# define GzipLibrary		-lz.dll
-#endif
 
 /* We don't need -lm */
 #define MathLibrary		/**/
@@ -115,7 +114,7 @@
  * /lib/libtermcap.a doesn't have tgetent, which is needed for
  * xc/programs/xterm/resize.exe
  */
-#define TermcapLibrary		-lncurses.dll
+#define TermcapLibrary		-lncurses
 
 #define AvoidNullMakeCommand	YES
 #define StripInstalledPrograms	YES
@@ -246,7 +246,7 @@
  * -DFD_SETSIZE=256 is needed for xfs and the xserver. cygwin defines the
  * FD_SETSIZE as 64 but some code expects it to be larger. 
  */ 
-#define StandardDefines		-D__i386__ -DWIN32_LEAN_AND_MEAN -DX_LOCALE \
+#define StandardDefines		-D__i386__ -DWIN32_LEAN_AND_MEAN \
 				-D_X86_ -D__CYGWIN__ -D_XOPEN_SOURCE \
 				-D_POSIX_C_SOURCE=199309L -D_BSD_SOURCE \
 				-D_SVID_SOURCE -D_GNU_SOURCE -DFD_SETSIZE=256 \
@@ -364,7 +372,6 @@
 #define XWinServerDefines	-DPIXPRIV \
 				-DXFree86Server \
 				-DX_BYTE_ORDER=ByteOrder \
-				XvExtensionDefines \
 				XWinXF86ConfigDefines XWinNativeGDIDefines \
 				XWinPrimaryFBDefines \
 				XWinEmulatePseudoDefines \
@@ -406,7 +404,7 @@
 # define BuildGlxExt		YES	/* Not enabled by default */
 #endif
 #ifndef BuildXprintLib
-# define BuildXprintLib		YES     /* Not enabled when Xprt disabled */
+# define BuildXprintLib		NO     /* Not enabled when Xprt disabled */
 #endif
 #ifndef BuildXterm
 # define BuildXterm		NO	/* Built separately on Cygwin */
--- origsrc/xorg-cf-files-1.0.2/cygwin.rules	2005-05-19 13:53:42.000000000 -0500
+++ src/xorg-cf-files-1.0.2/cygwin.rules	2008-11-04 10:58:30.540801300 -0600
@@ -22,11 +22,11 @@
 #define SharedLibXext		YES
 #define SharedLibXi		YES
 #define SharedLibXtst		YES
-#define SharedOldX		YES
-#define SharedLibXp		YES
+#define SharedOldX		NO
+#define SharedLibXp		NO
 #define SharedLibGlx		YES
-#define SharedLibDps		YES
-#define SharedLibDpsTk		YES
+#define SharedLibDps		NO
+#define SharedLibDpsTk		NO
 #define SharedLibGlu		YES
 #define SharedLibWindowsWM	NO
 #ifndef SharedDataSeparation
@@ -59,7 +59,7 @@
 Concat3(cyg,libname,-$(shell echo rev|sed s=\\..*==).dll)
 
 #define ImportLibraryName(libname, rev) \
-Concat3(lib,libname,-$(shell echo rev|sed s=\\..*==).dll.a)
+Concat3(lib,libname,.dll.a)
 
 #define ShortImportLibraryName(libname, rev) \
 Concat3(lib,libname,.dll.a)
@@ -233,8 +233,7 @@
 
 #define InstallSharedLibrary(libname,rev,dest)                          @@\
 InstallTarget(install,ImportLibraryName(libname,rev),$(INSTLIBFLAGS),$(USRLIBDIR)) @@\
-InstallTarget(install,SharedLibraryName(libname,rev),$(INSTBINFLAGS),$(BINDIR)) @@\
-InstallLink(install,ImportLibraryName(libname,rev),ShortImportLibraryName(libname,rev),$(USRLIBDIR))
+InstallTarget(install,SharedLibraryName(libname,rev),$(INSTBINFLAGS),$(BINDIR))
 
 
 /*
--- origsrc/xorg-cf-files-1.0.2/cygwin.tmpl	2005-05-19 13:53:42.000000000 -0500
+++ src/xorg-cf-files-1.0.2/cygwin.tmpl	2008-11-04 11:00:04.040801300 -0600
@@ -30,10 +30,7 @@
 #define SharedXResReqs 		$(LDPRELIB) $(XLIB)
 #define SharedXfontcacheReqs	$(LDPRELIB) $(XLIB)
 #define SharedXineramaReqs	$(LDPRELIB) $(XLIB)
-/*
- * We don't build Xss
- * #define SharedXssReqs	$(LDPRELIB) $(XLIB)
- */
+#define SharedXssReqs		$(LDPRELIB) $(XLIB)
 #define SharedXvMCReqs		$(LDPRELIB) $(XLIB)
 #define SharedXxf86dgaReqs	$(LDPRELIB) $(XLIB)
 #define SharedXxf86miscReqs	$(LDPRELIB) $(XLIB)
@@ -54,6 +51,10 @@
 XMULIB = $(XMULIBONLY) $(XTOOLLIB) $(XLIB)
 #endif
 
+#ifndef SharedXawRev
+# define SharedXawRev 7.0
+#endif
+
 #ifndef XawClientLibs
 # define XawClientLibs $(XAWLIB) $(XMULIBONLY) $(XTOOLLIB) $(XPMLIB) $(EXTRAXAWCLIENTLIBS) $(XLIB)
 #endif
