--- origsrc/xorg-cf-files-1.0.2/cygwin.cf	2005-07-06 04:47:49.000000000 -0500
+++ src/xorg-cf-files-1.0.2/cygwin.cf	2008-11-04 10:28:33.040801300 -0600
@@ -62,12 +62,14 @@
 # define OSTeenyVersion		DefaultOSTeenyVersion
 #endif
 
-#ifndef ProjectRoot
-# define ProjectRoot 		/usr/X11R6
+#ifndef XAppLoadDir
+# define XAppLoadDir       EtcX11Directory/app-defaults
 #endif
 
 #if !CrossCompiling
+#ifndef HasGcc2
 # define HasGcc2		YES
+#endif /* HasGcc2 */
 #endif /* !CrossCompiling */
 #define HasGcc2ForCplusplus	YES
 #define GccUsesGas		YES
@@ -98,15 +100,12 @@
  */
 #define Hastcl			YES
 #define Hastk			YES
-#define TclLibName		tcl80
-#define TkLibName		tk80
+#define TclLibName		tcl84
+#define TkLibName		tk84
 
 #define HasSockets		YES
 #define GzipFontCompression	YES
 #define HasZlib			YES
-#ifndef GzipLibrary
-# define GzipLibrary		-lz.dll
-#endif
 
 /* We don't need -lm */
 #define MathLibrary		/**/
@@ -115,7 +114,7 @@
  * /lib/libtermcap.a doesn't have tgetent, which is needed for
  * xc/programs/xterm/resize.exe
  */
-#define TermcapLibrary		-lncurses.dll
+#define TermcapLibrary		-lncurses
 
 #define AvoidNullMakeCommand	YES
 #define StripInstalledPrograms	YES
@@ -292,7 +291,7 @@
 # define BuildXWinMultiWindow	YES
 #endif
 #ifndef BuildXWinMultiWindowExtWM
-# define BuildXWinMultiWindowExtWM YES
+# define BuildXWinMultiWindowExtWM NO
 #endif
 #ifndef BuildXWinNativeGDI
 # define BuildXWinNativeGDI	NO
@@ -400,13 +399,13 @@
 # define BuildRandR		YES	/* Not supported by XWin Server */
 #endif
 #ifndef BuildXF86BigfontExt
-# define BuildXF86BigfontExt	YES	/* Not enabled by default */
+# define BuildXF86BigfontExt	NO	/* Not enabled by default */
 #endif
 #ifndef BuildGlxExt
 # define BuildGlxExt		YES	/* Not enabled by default */
 #endif
 #ifndef BuildXprintLib
-# define BuildXprintLib		YES     /* Not enabled when Xprt disabled */
+# define BuildXprintLib		NO     /* Not enabled when Xprt disabled */
 #endif
 #ifndef BuildXterm
 # define BuildXterm		NO	/* Built separately on Cygwin */
--- origsrc/xorg-cf-files-1.0.2/cygwin.rules	2005-05-19 13:53:42.000000000 -0500
+++ src/xorg-cf-files-1.0.2/cygwin.rules	2008-11-04 10:58:30.540801300 -0600
@@ -22,11 +22,11 @@
 #define SharedLibXext		YES
 #define SharedLibXi		YES
 #define SharedLibXtst		YES
-#define SharedOldX		YES
-#define SharedLibXp		YES
+#define SharedOldX		NO
+#define SharedLibXp		NO
 #define SharedLibGlx		YES
-#define SharedLibDps		YES
-#define SharedLibDpsTk		YES
+#define SharedLibDps		NO
+#define SharedLibDpsTk		NO
 #define SharedLibGlu		YES
 #define SharedLibWindowsWM	NO
 #ifndef SharedDataSeparation
@@ -59,7 +59,7 @@
 Concat3(cyg,libname,-$(shell echo rev|sed s=\\..*==).dll)
 
 #define ImportLibraryName(libname, rev) \
-Concat3(lib,libname,-$(shell echo rev|sed s=\\..*==).dll.a)
+Concat3(lib,libname,.dll.a)
 
 #define ShortImportLibraryName(libname, rev) \
 Concat3(lib,libname,.dll.a)
@@ -233,8 +233,7 @@
 
 #define InstallSharedLibrary(libname,rev,dest)                          @@\
 InstallTarget(install,ImportLibraryName(libname,rev),$(INSTLIBFLAGS),$(USRLIBDIR)) @@\
-InstallTarget(install,SharedLibraryName(libname,rev),$(INSTBINFLAGS),$(BINDIR)) @@\
-InstallLink(install,ImportLibraryName(libname,rev),ShortImportLibraryName(libname,rev),$(USRLIBDIR))
+InstallTarget(install,SharedLibraryName(libname,rev),$(INSTBINFLAGS),$(BINDIR))
 
 
 /*
--- origsrc/xorg-cf-files-1.0.2/cygwin.tmpl	2005-05-19 13:53:42.000000000 -0500
+++ src/xorg-cf-files-1.0.2/cygwin.tmpl	2008-11-04 11:00:04.040801300 -0600
@@ -30,10 +30,7 @@
 #define SharedXResReqs 		$(LDPRELIB) $(XLIB)
 #define SharedXfontcacheReqs	$(LDPRELIB) $(XLIB)
 #define SharedXineramaReqs	$(LDPRELIB) $(XLIB)
-/*
- * We don't build Xss
- * #define SharedXssReqs	$(LDPRELIB) $(XLIB)
- */
+#define SharedXssReqs		$(LDPRELIB) $(XLIB)
 #define SharedXvMCReqs		$(LDPRELIB) $(XLIB)
 #define SharedXxf86dgaReqs	$(LDPRELIB) $(XLIB)
 #define SharedXxf86miscReqs	$(LDPRELIB) $(XLIB)
@@ -54,6 +51,10 @@
 XMULIB = $(XMULIBONLY) $(XTOOLLIB) $(XLIB)
 #endif
 
+#ifndef SharedXawRev
+# define SharedXawRev 7.0
+#endif
+
 #ifndef XawClientLibs
 # define XawClientLibs $(XAWLIB) $(XMULIBONLY) $(XTOOLLIB) $(XPMLIB) $(EXTRAXAWCLIENTLIBS) $(XLIB)
 #endif
