--- origsrc/xorg-cf-files-1.0.2/cygwin.cf	2005-07-06 04:47:49.000000000 -0500
+++ src/xorg-cf-files-1.0.2/cygwin.cf	2006-08-17 18:37:38.021750000 -0500
@@ -62,12 +62,10 @@
 # define OSTeenyVersion		DefaultOSTeenyVersion
 #endif
 
-#ifndef ProjectRoot
-# define ProjectRoot 		/usr/X11R6
-#endif
-
 #if !CrossCompiling
+#ifndef HasGcc2
 # define HasGcc2		YES
+#endif /* HasGcc2 */
 #endif /* !CrossCompiling */
 #define HasGcc2ForCplusplus	YES
 #define GccUsesGas		YES
@@ -104,9 +102,6 @@
 #define HasSockets		YES
 #define GzipFontCompression	YES
 #define HasZlib			YES
-#ifndef GzipLibrary
-# define GzipLibrary		-lz.dll
-#endif
 
 /* We don't need -lm */
 #define MathLibrary		/**/
@@ -115,7 +110,7 @@
  * /lib/libtermcap.a doesn't have tgetent, which is needed for
  * xc/programs/xterm/resize.exe
  */
-#define TermcapLibrary		-lncurses.dll
+#define TermcapLibrary		-lncurses
 
 #define AvoidNullMakeCommand	YES
 #define StripInstalledPrograms	YES
@@ -406,7 +401,7 @@
 # define BuildGlxExt		YES	/* Not enabled by default */
 #endif
 #ifndef BuildXprintLib
-# define BuildXprintLib		YES     /* Not enabled when Xprt disabled */
+# define BuildXprintLib		NO     /* Not enabled when Xprt disabled */
 #endif
 #ifndef BuildXterm
 # define BuildXterm		NO	/* Built separately on Cygwin */
--- origsrc/xorg-cf-files-1.0.2/cygwin.rules	2005-05-19 13:53:42.000000000 -0500
+++ src/xorg-cf-files-1.0.2/cygwin.rules	2006-08-17 18:38:56.849875000 -0500
@@ -59,7 +59,7 @@
 Concat3(cyg,libname,-$(shell echo rev|sed s=\\..*==).dll)
 
 #define ImportLibraryName(libname, rev) \
-Concat3(lib,libname,-$(shell echo rev|sed s=\\..*==).dll.a)
+Concat3(lib,libname,.dll.a)
 
 #define ShortImportLibraryName(libname, rev) \
 Concat3(lib,libname,.dll.a)
@@ -233,8 +233,7 @@
 
 #define InstallSharedLibrary(libname,rev,dest)                          @@\
 InstallTarget(install,ImportLibraryName(libname,rev),$(INSTLIBFLAGS),$(USRLIBDIR)) @@\
-InstallTarget(install,SharedLibraryName(libname,rev),$(INSTBINFLAGS),$(BINDIR)) @@\
-InstallLink(install,ImportLibraryName(libname,rev),ShortImportLibraryName(libname,rev),$(USRLIBDIR))
+InstallTarget(install,SharedLibraryName(libname,rev),$(INSTBINFLAGS),$(BINDIR))
 
 
 /*
--- origsrc/xorg-cf-files-1.0.2/cygwin.tmpl	2005-05-19 13:53:42.000000000 -0500
+++ src/xorg-cf-files-1.0.2/cygwin.tmpl	2006-08-17 18:37:38.068625000 -0500
@@ -54,6 +54,10 @@
 XMULIB = $(XMULIBONLY) $(XTOOLLIB) $(XLIB)
 #endif
 
+XAWLIB = -lXaw7
+XAW7LIB = -lXaw7
+XAW6LIB = -lXaw6
+
 #ifndef XawClientLibs
 # define XawClientLibs $(XAWLIB) $(XMULIBONLY) $(XTOOLLIB) $(XPMLIB) $(EXTRAXAWCLIENTLIBS) $(XLIB)
 #endif
